canvas_modules/harness/test_resources/parameterDefs/slider_paramDef.json

```diff
@@ -0,0 +1,187 @@
+{
+  "titleDefinition": {
+    "title": "Slider with Min Max"
+  },
+  "current_parameters": {
+    "slider": 50,
+    "slider_double": 0.5,
+    "slider_error": 50,
+    "hide": true,
+    "slider-disable": false
+  },
+  "parameters": [
+    {
```

Comment:
Can you add more option? What happens if null or something undefined is passed in.  We usually add these into the parameterDef to validate.